.metrics.json

```diff
@@ -31,6 +31,15 @@
             "isPass":      "SIMULATION PASSED",
             "seed":        1
 	        },
+          {
+            "name":        "Previously generated riscv-dv test",
+            "build":       "uvmt_cv32",
+            "cmd":         "cd cv32/sim/uvmt_cv32; make asm ASM_PROG=riscv_ebreak_test_0 SIMULATOR=dsim USE_ISS=YES DSIM_WORK=/mux-flow/build/repo/dsim_work DSIM_RESULTS=/mux-flow/build/results",
+            "wavesCmd":    "cd cv32/sim/uvmt_cv32; make asm ASM_PROG=riscv_ebreak_test_0 SIMULATOR=dsim USE_ISS=YES DSIM_WORK=/mux-flow/build/repo/dsim_work DSIM_RESULTS=/mux-flow/build/results WAVES=1",
+            "metricsFile": "hello-world/metrics.db",
```

Comment:
metricsFile property is set incorrectly.  Looking at the artifacts associate with this test, the correct value should be:
`'metricsFile" : "riscv_ebreak_test_0/metrics.dv"`

Reply:
Thanks.  Fixes in.  Passes the Metrics' JSON lint check.